packages/expo-core/ios/EXCore/EXUtilities.m

```diff
@@ -116,4 +116,58 @@ + (CGFloat)screenScale
   return scale;
 }
 
+
+// Kind of copied from RN to make UIColor:(id)json work
++ (NSArray<NSNumber *> *)NSNumberArray:(id)json
+{
+  return json;
+}
+
++ (NSNumber *)NSNumber:(id)json
+{
+  return json;
+}
+
++ (CGFloat)CGFloat:(id)json
+{
+  return [[self NSNumber:json] floatValue];
+}
+
++ (NSInteger)NSInteger:(id)json
+{
+  return [[self NSNumber:json] integerValue];
+}
+
++ (NSUInteger)NSUInteger:(id)json
+{
+  return [[self NSNumber:json] unsignedIntegerValue];
+}
+
+// Copied from RN
++ (UIColor *)UIColor:(id)json
```

Comment:
not sure what the best approach here is.. so i went for the easiest and just copied this method over, along with a few others that it depends on for converting between types.

Reply:
There definitely is a need for some place for cross-unimodule utilities, like this `UIColor` conversion or [`FileSystemUtilities`](https://github.com/expo/expo/pull/3245#discussion_r250512163). However, I'm not certain `expo-core` is the place for such code… So far we were able to make `expo-core` contain code only responsible for universal modules architecture, maybe we should create another `expo-utilities` unimodule (like we did with `expo-errors`)? Or just put  helpers to `expo-core` as a "good enough" effort? What do you think, @bbarthec, @tsapeta?

Reply:
Yeah, I like the idea of separate `utilities` module as more and more modules depends on some small features. I wonder whether `expo-errors` might be included in such module?

Reply:
@sjchmiela @bbarthec - do you mind if we leave this in `expo-core` for the sake of this PR and in a followup we create `expo-utilities` or whatever and refactor this?